data/json/monsters/mammal.json

```diff
@@ -442,8 +447,9 @@
     "path_settings": { "max_dist": 10 },
     "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Fawn has size "SMALL", why give it harvest animal_medium? It needs to be a new category for animal_small.

Reply:
Again younger than I thought. Adjusting.